priv/gettext/default.pot

```diff
@@ -94,51 +94,3 @@ msgstr ""
 #: lib/transport_web/templates/page/login.html.eex:37
 msgid "login_what_is_going_to_happen"
```

Comment:
you don't put this one in a domain?

Reply:
Not in the scope of this PR, but I'll do in the next one.

Reply:
Take a look at the description of #77